src/views/HomeView.vue

```diff
@@ -65,12 +77,86 @@ const carouselSlides = [
     },
   ],
 ];
+
+const gameDetails = [
+  {
+    genres: [
+      { name: "Adventure" },
+      { name: "RPG" },
+      { name: "CO-OP" },
+      { name: "Horror" },
+    ],
+    platform: "apple",
+    imagePath: "Cyberpunk2077.jpg",
+    title: "CyberPunk 2077",
+    price: 159.99,
+    discountInPercentage: 0,
+    reviews: 2137,
+    star: 3,
+    description:
+      "Cyberpunk 2077 is an open-world, action-adventure RPG" +
+      " set in the dystopian metropolis of Night City, a place obsessed with power," +
+      ' glamour, and body modification. Players take on the role of "V" a mercenary' +
+      " outlaw whose life is upended after a heist goes wrong. V becomes entangled wit" +
+      "h a legendary rock star, Johnny Silverhand, whose digital ghost is now trapped inside" +
+      " V's mind via an experimental biochip. The main goal is to find a way to remove the" +
+      " chip and save V's life before it's overwritten, all while trying to build a legend in " +
+      "a world where corporate power is absolute and the line between human and machine " +
+      "is increasingly blurred.",
+  },
```

Comment:
The long description strings in gameDetails are built using string concatenation with +. This can be hard to read and format.

 Use JavaScript's template literals (backticks `) for multi-line strings. They are cleaner, preserve whitespace, and make the text much easier to read and maintain.

```suggestion
    description: `
      Cyberpunk 2077 is an open-world, action-adventure RPG
      set in the dystopian metropolis of Night City, a place obsessed with power,
      glamour, and body modification. (...)
    `
```